lib/src/cli.dart

```diff
@@ -15,16 +15,19 @@ import 'singleton.dart' as singleton;
 // This version must be updated in tandem with the pubspec version.
 const String appVersion = '0.8.5';
 
-List<String> grinderArgs() => _args;
-List<String> _args;
-bool _verifyProjectRoot;
+List<String> grinderArgs() {
+  var args = _args;
+  if (args == null) fail('grinderArgs() may only be called after grind().');
+  return args;
+}
+
+List<String>? _args;
 
 Future runTasks(
   List<String> args, {
   bool verifyProjectRoot = false,
 }) async {
-  _args = args ?? [];
-  _verifyProjectRoot = verifyProjectRoot;
+  _args = args;
```

Comment:
I don't know about actual uses in the wild, but it looks like this method was handling cases where the `args` param could be null.